sqldb/sqlc/migrations/000007_graph.up.sql

```diff
@@ -105,4 +105,8 @@ CREATE TABLE IF NOT EXISTS node_addresses (
 CREATE UNIQUE INDEX IF NOT EXISTS node_addresses_unique ON node_addresses (
     node_id, type, position
 );
-CREATE INDEX IF NOT EXISTS node_addresses_node_id_idx ON node_addresses(node_id);
\ No newline at end of file
+CREATE INDEX IF NOT EXISTS node_addresses_node_id_idx ON node_addresses(node_id);
+
+CREATE TABLE IF NOT EXISTS source_nodes (
+    node_id BIGINT PRIMARY KEY REFERENCES nodes (id) ON DELETE CASCADE
```

Comment:
nit: Needs to be `INTEGER PRIMARY KEY` for the Postgres schema replacement to work.

Reply:
> isnt that only the case if we want this to be auto incrementing?

Yes. But if you actually insert the key (and don't omit it in the `INSERT` statement), then I think it doesn't do any auto incrementation.

But yeah, I guess `UNIQUE` is less confusing.

Reply:
ok cool - will update to use UNIQUE 👍